plugins/ocp/ocp-nvme.c

```diff
@@ -3271,7 +3327,7 @@ static int get_c9_log_page(struct nvme_dev *dev, char *format)
 	int ret = 0;
 	__u8 *header_data;
 	struct telemetry_str_log_format *log_data;
-	nvme_print_flags_t fmt;
+	enum nvme_print_flags fmt;
```

Comment:
I suggest we keep this type as `nvme_print_flags_t` here. We just changed the printing flags type to this. We define the flags as enum type but then we want to be able to use several bits in variable. Thus this shouldn't be an enum anymore.

Reply:
@igaw 
Fixed. Please check.

I have fixed all the coding guideline issue. Patch failed due to below one. Even I have update commit description. Please check
-------------------------------------------------------------
[1/1] Check commit - c143e40863a693e58db373b89134fbfa5f4efb15
-------------------------------------------------------------
WARNING: Missing commit description - Add an appropriate one
Error: WARNING: Missing commit description - Add an appropriate one

Reply:
Your commit message needs to be edited. This has nothing to do what you edit in github's PR. checkpatch complains about the git commit itself.

`git commit --edit --amend` should open your text editor and you should fix this

![image](https://github.com/linux-nvme/nvme-cli/assets/1050803/14281340-b618-40f2-bd3c-8b0b25d7350b)